Code/RDGeneral/RDValue-doublemagic.h

```diff
@@ -411,6 +410,22 @@ inline bool rdvalue_cast<bool>(RDValue v) {
   throw boost::bad_any_cast();
 }
 
+// The RDValue has no getKey implementation
+//  so implement our own
+struct KeyIntPair {
+  int key;
```

Comment:
There is, but not much sadly, We have somewhere in the order of 20 bits maybe.  Might be okay for non-server work perhaps.